spatial-tracking-explainer.md

```diff
@@ -210,60 +210,54 @@ An `XRRay` object includes both an `origin` and `direction`, both given as `DOMP
 ### Application-supplied transforms
 Frequently developers will want to provide an additional, artificial transform on top of the user's tracked motion to allow the user to navigate larger virtual scenes than their tracking systems or physical space allows. This effect is traditionally accomplished by mathematically combining the API-provided transform with the desired additional application transforms. WebXR offers developers a simplification to ensure that all tracked values, such as viewer and input poses, are transformed consistently.
 
-Developers can specify application-specific transforms by setting the `originOffset` attribute of any `XRReferenceSpace`. The `originOffset` is initialized to an identity transform, and any values queried using the `XRReferenceSpace` will be offset by the `position` and `orientation` the `originOffset` describes. The `XRReferenceSpace`'s `originOffset` can be updated at any time and will immediately take effect, meaning that any subsequent poses queried with the `XRReferenceSpace` will take into account the new `originOffset`. Previously queried values will not be altered. Changing the `originOffset` between pose queries in a single frame is not advised, since it will cause inconsistencies in the tracking data and rendered output.
+Developers can specify application-specific transforms by calling the getOffsetReferenceSpace() method of any XRReferenceSpace. This returns a new XRReferenceSpace where the XRRigidTransform passed to getOffsetReferenceSpace() describes the position and orientation of the offset space's origin in relation to the base reference space's origin. Specifically, originOffset.position contains the coordinates of the new origin relative to the base reference space's origin. If the base reference space was also created with getOffsetReferenceSpace(), the overall offset is the combination of both transforms.
```

Comment:
> Seems strange to be calling out position, but not orientation. Almost implies that orientation is ignored.

That wasn't the intent, a reason for having this here is that it's a fairly straightforward example that's easy to sanity check in terms of coordinates, and the previous sentence already mentioned "position and orientation". It's difficult to make a short-but-concrete example based on quaternions.

We could add something along these lines, but I think this may get clunky?

... and originOffset.orientation is a quaternion representing a rotation that transforms vectors in offset space coordinates to their coordinate values in the base reference space.

Reply:
In general, if we could centrally establish once in the spec that a "pose" is a position and orientation applied in a specific order within a base space, we could easily and consistently describe all such use of poses throughout the spec in a straightforward manner:

> This returns a new XRReferenceSpace where the XRRigidTransform passed to getOffsetReferenceSpace() describes the pose of the offset space's origin within the base space.